src/main/java/javaClasses/homework_4/davit_balabekyan/math/Calculations.java

```diff
@@ -0,0 +1,36 @@
+package javaClasses.homework_4.davit_balabekyan.math;
+
+public class Calculations {
+
+    public static int getMean(int... intVarargs) {
+        int sum = 0;
+        for (int number : intVarargs) {
+            sum += number;
+        }
+        return sum / 3;
```

Comment:
In this case you'll have result only for 3 numbers. You should divide your sum, on number of the elements of the array, to get the arithmetical mean.

Reply:
Done)